src/MultiSelectDropdown/index.js

```diff
@@ -216,6 +217,8 @@ const MultiSelectDropdown = (props) => {
     size,
     color,
     theme,
+    addingOptionEnabled,
+    addOptionText,
```

Comment:
enableAdd and addOptionText should be added in propTypes and defaultProps. 
Default value for enableAdd is "false".
Default value for addOptionText is "Add new item".